bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/RawType.java

```diff
@@ -23,31 +23,51 @@
 public class RawType implements PrimitiveType, State {
 
     protected byte[] bytes;
+    protected String mimeType;
 
     public RawType() {
         this(new byte[0]);
     }
 
     public RawType(byte[] bytes) {
+        this(bytes, null);
+    }
+
+    public RawType(byte[] bytes, String mimeType) {
         this.bytes = bytes;
+        this.mimeType = mimeType;
     }
 
     public byte[] getBytes() {
         return bytes;
     }
 
+    public String getMimeType() {
+        return mimeType;
+    }
+
     public static RawType valueOf(String value) {
-        return new RawType(Base64.getDecoder().decode(value));
+        int idx;
+        if (value == null || !value.startsWith("data:") || ((idx = value.indexOf(",")) < 0)) {
+            return new RawType();
+        }
+        String mimeType = null;
+        int idx2 = value.indexOf(";");
+        if (idx2 > 5) {
+            mimeType = value.substring(5, idx2);
+        }
+        return new RawType(Base64.getDecoder().decode(value.substring(idx + 1)), mimeType);
     }
 
     @Override
     public String toString() {
-        return String.format("raw type: %d bytes", bytes.length);
+        return String.format("raw type (%s): %d bytes", (mimeType != null) ? mimeType : "unknown", bytes.length);

```

Comment:
`unknown` is not a valid media type. [below](https://github.com/eclipse/smarthome/pull/3055/files#diff-7a4c8bb7df8b8ef0705510067c5561f8R69) you have an empty string when `mimeType` is `null`, which according to [this](https://en.wikipedia.org/wiki/Data_URI_scheme#Syntax) is the same as `text/plain`.